.github/instructions/dsc-community-style-guidelines.instructions.md

```
@@ -23,6 +23,7 @@ applyTo: "**"
 - Integration tests: `tests/Integration/Commands/{CommandName}.Integration.Tests.ps1`
 
 ## Requirements
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)